lms/static/sass/features/_course-outline.scss

```diff
@@ -1,3 +1,9 @@
+.outline-header {
```

Comment:
There is no way for you to know this, but the course skeleton defines a page-title class that should be used for consistency:

https://outline-details.sandbox.edx.org/template/ux/reference/course-skeleton.html